app/src/main/java/com/example/bookclub/BookDetailsActivity.java

```diff
@@ -180,7 +212,7 @@ private Uri getLocalBitmapUri(ImageView shareImage) {
     }
 
     public void viewIntent(final String id) {
```

Comment:
I'd make this method a bit more descriptive, like `launchWebView` or something of the sort.